src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -31,9 +32,9 @@
  */
 public class TypicalPersons {
 
-    public static final Person ALICE = new PersonBuilder().withType("doctor").withNric("S1532142A")
+    public static final Doctor ALICE = new DoctorBuilder().withNric("S1532142A")
             .withName("Alice Pauline").withPhone("94351253").build();
-    public static final Person BENSON = new PersonBuilder().withType("doctor").withNric("T5231426Q")
+    public static final Doctor BENSON = new DoctorBuilder().withNric("T5231426Q")
```

Comment:
dont forget to change the `typicalPerson.json` file

Reply:
Its already doctor there, someone else changed it